### DISCUSSION TITLE
i have 3 questions to vladmandic

### DISCUSSION THREAD

**Discussion by IkariDevGIT** [vladmandic/sdnext#858] [Q&A]
1. when is Anapnoe ui coming(when will it be merged with this extension)?
2. when is the new extensions manager coming?
3. are you going to make your own extension "database"(it currently uses this json thing by auto1111)

**Comment by vladmandic** [comment]
re: 1 - ask him, he's actively working on it :)
re: 2 - i just completed it, a bit more testing, so probably tomorrow
re: 3 - sort-of. there is a github server action that collects extension information once-per-hour (including a1111) list into new auto-populated master list and client app fetches information from there instead from raw json on a1111 repo
